src/track/beatutils.cpp

```diff
@@ -241,7 +235,6 @@ QVector<double> BeatUtils::calculateIronedBeatMap(
     if (rawbeats.size() < kBeatsToCountTempo) {
         return rawbeats;
     }
-    // Length of a beat at globalBpm in mono samples.
     double secondsPerSample = 1 / static_cast<double>(sampleRate);
     double beatOffset = rawbeats[1];
```

Comment:
I think this is the ironedBeat, right? Can we rename it?

Reply:
No. This is the offset we add the mean beat lenght use to test the phase error.
CorrectedBeat is the ironedBeat, which I just renamed.
Do have suggestions for this name as well?